client/.gitignore

```diff
@@ -0,0 +1,3 @@
+TileInfo - Copie.json
```

Comment:
that's just a gitignore I put in client because I have some files I used for tests/debugging I didn't want to track them

This gitignore will be removed before merging the branch obv